Dockerfile.template

```diff
@@ -2,18 +2,18 @@
 FROM balenalib/%%BALENA_MACHINE_NAME%%-alpine-node:14-build as builder
 WORKDIR /usr/src
 
-RUN install_packages chromium
+# RUN install_packages chromium
```

Comment:
If we don't need them, let's get rid of the commented out stuff

Reply:
Yeah, I agree.